plugins/lookup/random_pet.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: random_pet
+    author:
+      - Abhijeet Kasurde (@Akasurde)
+    short_description: Generates random pet names
+    version_added: '3.1.0'
+    requirements:
+      - petname U(https://github.com/dustinkirkland/python-petname).
+    description:
+      - Generates random pet names that can be used as unique identifiers for the resources.
+    options:
+      words:
+        description:
+        - The number of words in the pet name.
+        default: 2
+        type: int
+      length:
+        description: The length of the pet name.
+        default: 6
+        type: int
+      prefix:
+        description: A string to prefix with the name.
+        type: str
+      separator:
+        description: The character to separate words in the pet name.
+        default: "-"
+        type: str
+'''
+
+EXAMPLES = r'''
+- name: Generate pet name
+  ansible.builtin.debug:
+    var: lookup('community.general.random_pet')
```

Comment:
How about adding an example output next to every example, like:
```suggestion
    var: lookup('community.general.random_pet')
    # Example result: 'lasting-toucan'
```
Same for the others. I think that makes it easier to understand what these options do, and what the lookup does in general.